api/graphql/schema-types-defs/Verse.graphql

```diff
@@ -0,0 +1,46 @@
+type visraamObject {
+  p: Int
```

Comment:
sometimes the positions comes back as a string (very rarely, but happens in japji sahib a couple times; its enough to where you have to count for it)

Reply:
examples?

Reply:
I see 
![image](https://user-images.githubusercontent.com/2982784/86523007-f2ca6900-be1a-11ea-821f-41ad4f1e4eca.png)

Reply:
looks like it does the conversion 
![image](https://user-images.githubusercontent.com/2982784/86523039-4937a780-be1b-11ea-8a67-ba245ee24be6.png)

Reply:
fixed the "t" prop